claim_batch/services.py

```diff
@@ -431,6 +432,207 @@ def _get_capitation_region_and_district(location_id):
 def do_process_batch(audit_user_id, location_id, period, year):
     processed_ids = set()  # As we update claims, we add the claims not in relative pricing and then update the status
```

Comment:
@delcroip Is this part going to be used in new batch run design? In old method this is used.